scripts/apps/search/views/search-parameters.html

```diff
@@ -162,16 +169,46 @@
             </div>
         </div>
 
+        <div ng-if="search_config.date_published"
```

Comment:
There's a loop rendering multiple fields here, so you can't use `ng-if="search_config.date_published"`, because the code isn't specific to this single field. Instead, you should make sure that `dateFilters` is filtered before the view is rendered.